aimmo-game/tests/test_service.py

```diff
@@ -1,26 +1,87 @@
 from __future__ import absolute_import
 
-from unittest import TestCase
+from unittest import TestCase, skip
 
 from simulation.game_state import GameState
 from simulation.location import Location
 from simulation.turn_manager import state_provider
 from simulation.world_map import WorldMap
 from simulation.world_state import WorldState
+from simulation.avatar.avatar_manager import AvatarManager
 
 import service
 
 from .test_simulation.dummy_avatar import MoveEastDummy
 from .test_simulation.maps import MockPickup
 from .test_simulation.test_world_map import MockCell
 
-class SimpleAvatarManager(object):
-    avatars = [MoveEastDummy(1, Location(0, -1))]
+class SimpleAvatarManager(AvatarManager):
+    def __init__(self):
+       super(SimpleAvatarManager, self).__init__()
 
-# TODO: Write test for the new API...
-class TestService(TestCase):
-    def test_healthy(self):
+       avatar = MoveEastDummy(1, Location(0, -1))
+       self.avatars_by_id = {
+           1 : avatar
+       }
+       self.avatars_to_create_by_id = {
+           1 : avatar
```

Comment:
it's the same in to_create and in created, is this relevant?

Reply:
I think it does not affect the test, but is confusing, so I will modify it.